salt/metalk8s/addons/ui/config/metalk8s-ui-config.yaml.j2

```diff
@@ -4,7 +4,20 @@
 apiVersion: addons.metalk8s.scality.com/v1alpha2
 kind: UIConfig
 spec:
+  # Deployment configuration
+  deployment:
+    replicas: 2
+    affinity:
+      podAntiAffinity:
+        soft:
+          - topologyKey: kubernetes.io/hostname
+        #   - topologyKey: my.second.important/topologyKey
+        #     weight: 42
+        # hard:
+        #   - topologyKey: kubernetes.io/hostname
```

Comment:
nit: why do we keep those commented lines ?

Reply:
It's because we take this file in the documentation so that from the documentation the user can see those "example"